operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

```diff
@@ -559,7 +559,15 @@ Verify that the Lustre file system is available from the management cluster.
     +----------------+------+--------+-------+------+---------+------+-------+------------+----------+
     ```
 
-1. To check the health and status of the management cluster after a power cycle, refer to the "Platform Health Checks" section in [Validate CSM Health](../validate_csm_health.md).
+1. Check the health and status of the management cluster after a power cycle.
+
+    1. Enable the `goss-servers` on all the NCNs. For example:
+
+        ```bash
+        ncn# pdsh -w ncn-m00[1-3],ncn-w00[1-3],ncn-s00[1-3] systemctl enable --now goss-servers.service
```

Comment:
This shouldn't hardcode to just the 9 NCNs, it should fetch all of them.

Reply:
Agreed. We have other places in the docs where we just get them from /etc/hosts using grep

Reply:
I was making this consistent with the existing doc - I will see how that is done and update the doc